language/src/ceylon/language/Sequence.ceylon

```diff
@@ -287,8 +287,13 @@ shared sealed interface Sequence<out Element=Anything>
         
         first => outer.first;
         size => outer.size*times;
-        rest => sublistFrom(1).sequence(); //TODO!
-        
+        rest => if (times==1)
+                    then outer.rest
+                    else let (tail = Repeat(times-1))
+                         if (outer.longerThan(1))
+                             then outer.rest.append(tail)
+                             else tail;
+
         getFromFirst(Integer index)
```

Comment:
So original 'rest' implementation fails on the JVM because of a bug resolving 'outer'...
This means my solution do also works because of the same bug, and will break if bug is fixed!
🤦‍
Replacing `Repeat(times-1)` by `outer.Repeat(times-1)` will do the trick, so the same code will work with or without fixing the 'outer' bug... but then, I wonder if original code was better...